session_state.go

```diff
@@ -75,20 +76,39 @@ func (sm *stateMachine) Disconnected(session *session) {
 	}
 }
 
+var msgTypeExecutionReport = []byte("\x0135=8\x01")
+
+func IsExecutionReport(m []byte) bool {
```

Comment:
perhaps these belong around here? https://github.com/alpacahq/quickfix/blob/8a2144a438445349de04c07d02fbb8bcb40df063/msg_type.go#L26

and avoid exporting unless required